noxfile.py

```diff
@@ -180,3 +181,64 @@ def make(session: nox.Session):
         *(args.make_args or ("clean", "coredocs")),
     ]
     session.run("make", "-C", "docs/docsite", *make_args, external=True)
+
+
+def _rm_tmpdir(tmpdir):
+    rmtree(tmpdir, ignore_errors=True)
+
+
+@nox.session
+def tag(session: nox.Session):
+    """
+    Check the core repo for new releases and create tags in ansible-documentation
+    """
+    install(session, req="tag")
+
+    CORE_REPO = "https://github.com/ansible/ansible"
+    DOCS_REPO = "https://github.com/ansible/ansible-documentation"
+
+    args = list(session.posargs)
+    tmpdir = None
```

Comment:
No need to initialize tmpdir to None.

Reply:
May I ask why? This is probably my fault so I'd like to understand. Cheers.

Reply:
I think the comment only makes sense if `_rm_tmpdir` is removed (as also suggested by @gotmax23). If `_rm_tmpdir` and the call to it are kept, this initialization needs to be kept as well.